src/manifest/assertions/IngredientAssertion.ts

```diff
@@ -74,6 +78,31 @@ export class IngredientAssertion extends Assertion {
     public validationStatus?: ValidationStatusCode[];
     public c2pa_manifest?: HashedURI;
 
+    public static new(title: string, format: string, instanceId: string, documentId?: string): IngredientAssertion {
```

Comment:
I'm not sure I'm a fan of the `new` / `newV2` method names here. I think it's easily confused with `new IngredientAssertion()`. In other places we use `create` static methods instead.

Also please add JSDoc :)